src/componets/chatBotBody/chatBotBody.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
I think a better folders structure would be like
`src/components/ChatBot/ChatBody.tsx`